src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -24,15 +24,30 @@
     = render partial: 'webui2/shared/buildresult_box', locals: { project: @project.name }
   .list
     .card
-      %h5.card-header
-        Packages
-        = "(#{@packages.length})"
-      = render partial: 'project_packages'
+      .bg-light
+        %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap{ role: 'tablist', id: 'packages-tabs' }
```

Comment:
IDs and classes are written directly in HAML when they are static. So this line would then be:

%ul.nav.nav-tabs.pt-2.px-3.flex-nowrap<b>#packages-tab</b>{ role: 'tablist' }

Rubocop might complain about the order of classes and the ID, then switch them if needed. I don't remember by heart what we decided to use.